relay-general/src/store/normalize/span/tag_extraction.rs

```diff
@@ -219,10 +218,19 @@ pub(crate) fn extract_tags(span: &Span, config: &Config) -> BTreeMap<SpanTagKey,
         }
 
         let domain = if span_op == "http.client" {
-            span.description
-                .value()
-                .and_then(|url| domain_from_http_url(url))
-                .map(|d| d.to_lowercase())
+            // HACK: Parse the normalized description to get the normalized domain.
+            scrubbed_description
+                .and_then(|d| d.split_once(' '))
+                .and_then(|(_, d)| Url::parse(d).ok())
+                .and_then(|url| {
+                    url.domain().map(|d| {
```

Comment:
This code repeats a bit in `scrub_http`  function. Maybe it makes sense to create a smallish helper util for this. But it's also fine to keep it this way.